lib/rules/web/admin_console/4.7/common_ui_elements.xyaml

```diff
@@ -411,7 +411,7 @@ click_kebab_of_one_resource:
   - selector:
       xpath: //*[text()='<resource_name>']//ancestor::tr//button[@data-test-id='kebab-button']
     op: click
-    timeout: 20
+    timeout: 40
```

Comment:
this has been updated in my another pr https://github.com/openshift/verification-tests/pull/1799/files which I guess the timeout should not matter....